### PR TITLE
feat(pagination): change pagination display size (sm/lg)

### LINKED ISSUE #314: Pagination: support "size" input
We should introduce the `size` `@Input` that could take the following values: `sm` or `lg` (empty by default). When this input is set to something else than empty we should add `size-sm` or `size-lg` class. This is very similar to what we do for a modal window here: https://github.com/ng-bootstrap/core/blob/fb4d331443f48321826d09a2292d3bd68ae21168/src/modal/modal_window.ts#L15

Ref: http://v4-alpha.getbootstrap.com/components/pagination/#sizing

### PR DESCRIPTION
**Pull request by maxokorokov**
Adding input 'size' to pagination component to support bootstrap 'pagination-lg' and 'pagination-sm' styles

Closes #314

### CODE REVIEW

File: src/pagination/pagination.spec.ts [hunk 1]
```diff
@@ -243,6 +247,40 @@ describe('ngb-pagination', () => {
...
+           fixture.detectChanges();
+           expect(classList.length).toBe(2);
+           expect(classList).toContain('pagination');
+           expect(classList).toContain('pagination-lg');
+
+           // removing large size
+           fixture.componentInstance.size = '';
+           fixture.detectChanges();
+           expect(classList.length).toBe(1);
+           expect(classList).toContain('pagination');
```
Comment (pkozlowski-opensource) [inline src/pagination/pagination.spec.ts]: Please add `expect(classList).not.toContain('pagination-lg');`

Comment (maxokorokov) [inline src/pagination/pagination.spec.ts]: Hmm, you don't like checking length to be 1?

**Comment by pkozlowski-opensource** [comment]
LGTM part from the remark about tests.

**Comment by maxokorokov** [comment]
Thanks for the explanation, amended

**Comment by pkozlowski-opensource** [comment]
LGTM. Will merge as soon as TravisCI is green.

**Pull request closed without merging**